### ISSUE TITLE
Fading to orange

### ISSUE THREAD

**Issue by dinodona** [fabsenet/adrilight#77]
Hi Fabsenet,

I recently changed my config for more LEDs in total ( 206 > 254 )
I didn't do the circuit the same way and now, I'm having this "issue" with white led replaced by orange colour although my power supply should be sufficient.
It is occuring at the middle portion of the entire strip.

Can you post a scheme to explain how you had more power points from the power supply itself in order to avoid voltadge drop etc... 

Basically, I just powered the LED strip appart from the Arduino, as described in the scheme above :

![image](https://user-images.githubusercontent.com/31877266/51249847-e76c4f00-1994-11e9-8805-59d1dd19f00d.png)

Config > WS2812B / 5V-15A Power Supply / Arduino Uno

**Comment by psbeau** [comment]
Link the GNDs of your 5V power supplies

**Comment by dinodona** [comment]
How it could affect voltage drop / amperage loss in the LED strip ?
I don't get it

**Comment by psbeau** [comment]
I don't know, but if your sketch is correct - 15A power supply, thick gauge wire connected at both ends, then there shouldn't be a power issue. I then looked at what's wrong with the sketch. What gauge wire are you using?
I have 414 LEDs, 15A power supply, GNDs connected together, 2.5 sq mm wire connected at both ends and I don't have a problem with whites displaying correctly.

**Comment by psbeau** [comment]
I don't know how this works but are you limiting the brightness in adrilight.ino?

**Comment by dinodona** [comment]
Nope ! Adrilight.ino is nearly the default one
M'y power supply is only connected at the beginning of the strip.
Should I divide it at the "source" and connect it at the end of the strip also ?

**Comment by dinodona** [comment]
It does the trick (;
Thx @psbeau

**Comment by fabsenet** [comment]
great, looks like I am late to this party.

For reference:

- try to light up only some LEDs at a time to make sure, the yellow color is really caused by too much power loss
- connect the power to both ends (as you did)
- connect power at multiple points in between! you can soulder the power wires to the points, were you could also cut your strip. you only need to make sure to NOT provide more than a single data pin.

@psbeau ...are you watching the repo? thanks for the help!
